build-system/tasks/build.js

```diff
@@ -146,6 +146,11 @@ build.flags = {
 
 watch.description = 'Watches for changes in files, re-builds when detected';
 watch.flags = {
+  with_inabox: '  Also watch and build the amp-inabox.js binary.',
+  with_shadow: '  Also watch and build the amp-shadow.js binary.',
+  with_video_iframe_integration:
+    '  Also watch and build the video-iframe-integration.js binary.',
+  with_amp_story_embed: '  Also watch and build the amp-story-embed.js binary.',
```

Comment:
These flags no longer exist. Can you sync forward and resolve?

Also, where does `argv. with_amp_story_embed` get used? If the file is automatically picked up during build, you can just delete the flag.

Reply:
(see other comment)